File: 1134.py
al = 0
ga = 0
di = 0
x = 0
while x != 4:
    x = int(input())
    if x == 1:
        al = al + 1
    if x == 2:
        ga = ga + 1
    if x == 3:
        di = di + 1
print('MUITO OBRIGADO')
print('Alcool: {}'.format(al))
print('Gasolina: {}'.format(ga))
print('Diesel: {}'.format(di))

File: CSharp/1134.cs
using System; 

class URI {

    static void Main(string[] args) 
    {
        char x;
        int a=0;
        int g=0;
        int d=0;
        bool brecar = true;

        while(brecar)
        {
            x = char.Parse(Console.ReadLine());
            switch(x)
            {
                case '1':
                    a++;
                    break;

                case '2':
                    g++;
                    break;

                case '3':
                    d++;
                    break;
                
                case '4':
                    brecar=false;
                    break;

                default:
                    break;
            }
        }
        Console.WriteLine("MUITO OBRIGADO");
        Console.WriteLine($"Alcool: {a}");
        Console.WriteLine($"Gasolina: {g}");
        Console.WriteLine($"Diesel: {d}");

    }
}